packages/ui/Toggleable/tests/Toggleable-specs.js

```diff
@@ -149,11 +149,11 @@ describe('Toggleable', () => {
 		);
 
 		test(
-			'should use selected prop when selected changed from truthy to undefined',
+			'should use defaultSelected prop when selected changed from truthy to undefined',
```

Comment:
I've changed the behavior intentionally so it matches the test above in which `Toggleable` uses `defaultSelected` when `selected` is `null` at mount time.

Reply:
This seems like a pretty significant change.  Do we want to allow toggling from controlled to uncontrolled?

Reply:
That's not really what is happening here. It would still be considered controlled because the initial state included the `selected` prop. The change is only that for Toggleable (per the test above), `null` is treated as "unset" so it reverts to use the default value in that case.

Reply:
It seems like that's exactly what's happening here.  The component, already mounted, reverts to `defaultSelected` which should not apply for a controlled component, after `selected` becomes `undefined`.  Am I misreading the test?

Reply:
This is my mistake. I commented on the wrong test initially. It was intended for the change above.

To your point, this is a notable change that we should discuss further. Sorry for the confusion I caused!